src/scss/_post-single.scss

```diff
@@ -0,0 +1,13 @@
+.featured-image {
+  display: block !important;
+  width: 100% !important;
+
+  text-align: center !important;
+
+  }
+  .featured-image img {
+  display: inline-block !important;
+  height: 500px !important;
```

Comment:
Usually, we use `min-height` to set the height on these, and adjust things a little on mobile.

Reply:
height was changed to min-height